app/src/main/res/layout/prompt_alert.xml

```diff
@@ -1,73 +1,84 @@
+    android:background="@drawable/prompt_background">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/layout"
-        android:layout_width="match_parent"
+    <LinearLayout
+        android:id="@+id/layoutAlertMessages"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:background="@drawable/prompt_background">
+        android:orientation="vertical"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/positiveButton">
 
         <TextView
             android:id="@+id/alertTitle"
-            android:layout_width="0dp"
-            android:layout_height="match_parent"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
             android:layout_marginStart="20dp"
             android:layout_marginTop="10dp"
             android:layout_marginEnd="20dp"
+            android:layout_weight="7"
             android:gravity="center_horizontal"
             android:textSize="24sp"
             android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@id/bodyContainer"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             tools:text="Title" />
+
         <com.igalia.wolvic.ui.views.CustomScrollView
             android:id="@+id/bodyContainer"
             style="@style/customScrollViewStyle"
-            android:fastScrollAlwaysVisible="false"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:layout_centerHorizontal="true"
+            android:layout_height="0dp"
+            android:layout_marginBottom="10dp"
+            android:layout_weight="10"
+            android:fastScrollAlwaysVisible="false"
             app:layout_constraintBottom_toTopOf="@+id/positiveButton"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/alertTitle"
-            android:layout_marginBottom="10dp">
-        <TextView
-            android:id="@+id/alertMessage"
-            android:layout_marginStart="20dp"
-            android:layout_marginTop="10dp"
-            android:layout_marginEnd="20dp"
-            android:layout_marginBottom="10dp"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:gravity="center_horizontal"
-            android:maxHeight="@dimen/prompt_content_max_height"
-            android:scrollbars="vertical"
-            android:textSize="18sp"
-            tools:text="Message" />
+            app:layout_constraintTop_toBottomOf="@+id/alertTitle">
+
+            <TextView
```

Comment:
Unexpected extra indentation in all these lines

Reply:
Hello @svillar :)
I usually do a ctrl+alt+L --refactor code for the file i am working on.
I think the indentation is caused by that.
Is it ok to leave them as is?
For future contributions should i avoid Ctrl+Alt+L - shortcut to refactor the code ?

Reply:
Yeah in general changing indentation is not a good idea because when examining the history of changes in git, you'd have those changes that do not alter the code but just the indentation. And those changes make the process of determining who changed what more difficult. You can leave this as is, not a big deal.